mappings/net/minecraft/world/entity/EntityIdMapSet.mapping

```diff
@@ -0,0 +1,6 @@
+CLASS net/minecraft/class_5578 net/minecraft/world/entity/EntityIdMapSet
```

Comment:
`MapSet` is a very confusing name. Proposed `Sectioned` prefix as `SectionedEntityIdMap` in the last PR.

Reply:
Honestly all those idmap, idset, idmapset are poor names. I took them from the previous contributors without looking for alternatives. Btw if you don't mind I hope you can peek at https://github.com/FabricMC/Enigma/pull/338 real quick

Reply:
So now my proposal is like this:
`class_5578`: `EntityLookup` (old `EntityIdMap`)
`class_5570`: `EntityIndex` (old `EntitySet`)
`class_5573`: `EntityVoxels`, `EntitySections`, or something (old `SectionedEntityCache`)

Reply:
Decided to keep `SectionedEntityCache` but changed the other two. `Cache` might mean like storage in general, though in programming it often means expiring temporary storage